contribute.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "MobilePartners",
+    "description": "mobilepartners.mozilla.org is the self service website for Firefox OS OEMs and Operators.",
+    "repository": {
+        "url": "https://github.com/mozilla/fxoss",
+        "license": "MPL 2.0",
+        "tests": "https://travis-ci.org/mozilla/fxoss"
+    },
+    "bugs": {
+        "list": "https://github.com/mozilla/fxoss/issues",
+        "report": "https://github.com/mozilla/fxoss/issues/new"
```

Comment:
We have some old issues from when this project was primarily developed by Caktus that we need to clean up, but once their contract ended we switched to bugzilla, so a better link for the list would probably be: https://bugzilla.mozilla.org/buglist.cgi?component=mobilepartners.mozilla.org&product=Websites&bug_status=__open__&list_id=12070711 and to report new bugs: https://bugzilla.mozilla.org/enter_bug.cgi?product=Websites&component=mobilepartners.mozilla.org